packages/create-vite/index.js

```diff
@@ -3,7 +3,9 @@
 // @ts-check
 const fs = require('fs')
 const path = require('path')
-const argv = require('minimist')(process.argv.slice(2))
+// For solving the problem failed to create a project with number-only argument,
+// the second argument of "minimist" is added to convert it to string type.
```

Comment:
To me, it would have been helpful to know what `_` was referring to. What do you think of something like this? We can also add the PR number so people can check here for more information
```suggestion
// Avoids autoconversion to number of the project name by defining that the args 
// non associated with an option ( _ ) needs to be parsed as a string. See #4606
```

Reply:
Thanks, @patak-js !

I think it helps a user a lot. I'll adopt your opinion.